### PR TITLE
Merge wiki home page into readme

### PR DESCRIPTION
**Pull request by mikemahoney218**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,29 +2,49 @@
...
 
-## What’s Wrong
 
-If you ever got convicted of a crime or arraigned, even if it turns out someone made a mistake, you’ll have a criminal record. Anyone that does a background check, like an employer, sees that you have a criminal record and that’s the end of it. You lose out.
+## What’s Wrong
 
-**Even something false or small can stop you from getting honest work, housing, education, and other opportunities.**
+Each year, thousands of Massachusetts children and teenagers become involved with the criminal justice system. In Massachusetts, children between the ages of 12 and 18 may be arrested, charged in court, removed from their homes, and confined in secure, locked facilities. Once released, they may not be able to secure employment or housing due to their criminal records -- even if they were never convicted in the first place.
```
Comment (knod) [inline README.md:15]: "Once released" doesn't match up with "never convicted". Just noticed that.

File: README.md [hunk 2]
```diff
@@ -2,29 +2,49 @@
 
 # Clean Slate
 
-_Expungement with Citizens for Juvenile Justice, Code for Boston, and legal aid services._
+_Changing the system and changing lives._
```
Comment (knod) [inline README.md:5]: We may want to make this a bit less grandiose.

File: README.md [hunk 3]
```diff
@@ -2,29 +2,49 @@
...
+## What’s Wrong
 
-**Even something false or small can stop you from getting honest work, housing, education, and other opportunities.**
+Each year, thousands of Massachusetts children and teenagers become involved with the criminal justice system. In Massachusetts, children between the ages of 12 and 18 may be arrested, charged in court, removed from their homes, and confined in secure, locked facilities. Once released, they may not be able to secure employment or housing due to their criminal records -- even if they were never convicted in the first place.
 
-Massachusetts expungement laws were created in October of 2018, but almost no one can qualify and applying is hard. Sometimes it’s even a bad idea to expunge your records and people don’t know.
+People may be able to clear their records under MA G.L. c.276, § 100E-100U, a 239-part law passed in 2018 allowing sealing or expungement of certain criminal records that occurred before a person's 21st birthday. But the law is extremely complicated, with hundreds of criteria that may disqualify an individual. As such, very few people have been able to improve their prospects through the law.
```
Comment (knod) [inline README.md:17]: The law in Oct. 2018 created expungement. I believe sealing existed before that. Also see note about possibly not going into sealing on this page.

File: README.md [hunk 4]
```diff
@@ -2,29 +2,49 @@
...
 # Clean Slate
 
-_Expungement with Citizens for Juvenile Justice, Code for Boston, and legal aid services._
+_Changing the system and changing lives._
 
-**Expungement:** "Having a criminal record expunged means that the record will be permanently destroyed so that it's no longer accessible by the court or any other state, municipal, or county agencies." - https://www.mass.gov/expunge-your-criminal-record
+## Terms to Know
 
-_Note: Expungement is different than sealing_
+**Expungement:** "Having a criminal record expunged means that the record will be permanently destroyed so that it's no longer accessible by the court or any other state, municipal, or county agencies." ([source](https://www.mass.gov/expunge-your-criminal-record))
```
Comment (knod) [inline README.md:9]: Would like to rethink using this quote, since it's not really how the situation pans out right now.

**Comment by berit** [comment]
Is this PR meant to address issue #74 ?

**Comment by linnal86** [comment]
These are good updates but will not merge due to conflicts, may revisit and add to README but not via this pull request.

**Pull request closed without merging**